Marlin/Configuration.h

```diff
@@ -589,6 +589,8 @@ const bool Z_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the logic
 //=============================================================================
 
 // @section more
+// RigidBot - Disable stepper drivers until voltage has stabilized
+//#define STEPPER_RESET_FIX
```

Comment:
This name should probably be more descriptive about what it actually does, and perhaps it belongs in `Configuration_adv.h` instead. Or, it may be enough just to check for the presence of `STEPPER_RESET_PIN` adding a line like `#define HAS_STEPPER_RESET (PIN_EXISTS(STEPPER_RESET))` (to `Conditionals.h`) and drop this extra configuration option altogether.